### PR TITLE
feat(routes): Fix for rendering routes in a map and minor style adjustments.

### PR DESCRIPTION
**Pull request by inesluparia**

### CODE REVIEW

File: website/src/pages/plants/plantsPage.js [hunk 1]
```diff
@@ -9,36 +9,35 @@ function PlantsPage(props) {
...
+            <ul id="list-container">
+                ${plants.map(plant => `
+                <li data-plant-id=${plant.id}>
+                            ${plant.name}, ${plant.latinName}
+                </li>`).join("")}
+            </ul>  
+        </section>
+        <hr> 
         <aside id="details-container">
             <h2>${plants[2].name}</h2>
```
Comment (davpfre) [inline website/src/pages/plants/plantsPage.js:35]: Hardcoded index values? Just making sure that this is intentional.

File: website/src/pages/routes/routesPage.js [hunk 2]
```diff
@@ -55,25 +51,32 @@ function RoutesPage(props) {
...
+    })
 
-    async function loadRoute (...coords) {
-        map.resize();
-        const waypoints = await mapService.getRouteFromCoordinates(
-            [15.188356982912637, 55.320417209601885], [15.1928236, 55.3201917]
+    async function loadRoute (coordinates) {
+        //map.resize();
+        const waypoints = await getRouteFromCoordinates(
+            coordinates
```
Comment (davpfre) [inline website/src/pages/routes/routesPage.js]: Using the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax), each element in the array can be spread out as individual parameters for the function.
```suggestion
            ...coordinates
```

File: website/src/services/mapService.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ export function createMap(rootElement) {
  *
  * @returns {Promise<Waypoint>} Returns a promise that resolves to an array of {@link Waypoint}.
  */
-export function getRouteFromCoordinates(...coordinates) {
+export function getRouteFromCoordinates(coordinates) {
```
Comment (davpfre) [inline website/src/services/mapService.js]:
```suggestion
export function getRouteFromCoordinates(...coordinates) {
```

**Review by davpfre — DISMISSED** [review]
Generally fine, though some rippling changes. Use of the spread syntax is highly preferred, so I'd suggest not to do away with it.

On another note `#page-container` is being styled in every .scss file, though it needn't be. All styles are imported upon loading the website, so having it in any one of the stylesheets, is sufficient. Though it be nice to keep global styles in the style.scss file.

File: website/src/pages/plants/plantsPage.js (hunk 1 above)
Comment (inesluparia) [reply website/src/pages/plants/plantsPage.js:35]: It is intentional. Want to make sure it renders a nice plant in the details container the first time it loads.

File: website/src/pages/routes/routesPage.js (hunk 2 above)
Comment (inesluparia) [reply website/src/pages/routes/routesPage.js]: Your suggestion breaks the code. As we already talked about, it is just the fix we found with Asger to make the code work!

File: website/src/services/mapService.js (hunk 3 above)
Comment (inesluparia) [reply website/src/services/mapService.js]: Same here...

**Comment by inesluparia** [comment]
It think we can called this resolved after our conversation? I tried but I could keep your preference in syntax, and since it's just a preference... Regarding #page-container, it has always been in only one place so there's actually no issue there..

**Comment by inesluparia** [comment]
*could NOT

**Comment by inesluparia** [comment]
David, I love that you added drawMarkerOnMap though I wouldn't have done it calling a conflinct. I actually cant figure out how to accept it from here, so maybe you can?

**Comment by davpfre** [comment]
I don't get what you're talking about. Once conflicts are fixed, it can be merged.

**Comment by inesluparia** [comment]
I cannot accept or change the 'conflicting file' where you add DrawMarkerOnMap, neither from Gtihub nor they show up in intelliJ so I don't know how to fix it...

**Pull request merged**